.github/workflows/prepare-release.yml

```
@@ -0,0 +1,23 @@
+name: Prepare Buildpack Releases
```

(this repo only has one buildpack, and that's unlikely to change)

```suggestion
name: Prepare Buildpack Release
```